### ISSUE TITLE
Issue arised when a product add to compare with another product

### ISSUE THREAD

**Issue by rahul-ks176** [magento/magento2#29691]
### Preconditions (*)

1.Magento ver. 2.3.5-p2
2.PHP 7.3.20
3.mysql  Ver 14.14

### Steps to reproduce (*)

1.Adding two simple products from the admin->catalog->products section.
2.adding these two products add to the compare list.
3.compare these two added products.

### Expected result (*)

1. The CSS of the compare list page should not be broken and links should be clickable. 

### Actual result (*)

1.CSS is broken of the compare list page and available links also not worked on that page.

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [x] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [x] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [x] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._
![compare product](https://user-images.githubusercontent.com/61263875/90722317-3d734800-e2d8-11ea-87e2-05ec3da38c8e.png)

**Comment by rahul-ks176** [comment]
We add the two products from admin section. after adding those products add to the compare list, we compare these two products then the CSS is broken of compare list page

**Comment by krishprakash** [comment]
@magento give me 2.4-develop instance

**Comment by magento-engcom-team** [comment]
Hi @krishprakash. Thank you for your request. I'm working on Magento 2.4-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @krishprakash, here is your [Magento instance](https://i-29691-2-4-develop.instances.magento-community.engineering).
Admin access: [https://i-29691-2-4-develop.instances.magento-community.engineering/admin_82df](https://i-29691-2-4-develop.instances.magento-community.engineering/admin_82df)
Login: `dad1b474` Password: `37afb44e14da`
Instance will be terminated in up to 3 hours.

**Comment by krishprakash** [comment]
Hi @rahul-ks176. I'm not able to reproduce the issue on latest instance 
Kindly provide the additional steps OR any specific case to reproduce the issue 
Refer Screenshot:
![image](https://user-images.githubusercontent.com/46617848/90763942-2e56bf00-e305-11ea-9699-88d333a9a733.png)

**Comment by rahul-ks176** [comment]
Ok, thanks @krishprakash <https://github.com/krishprakash>,

Actually this issue arises in Magento ver 2.3.5-p2 which I am using at this
time.
I have also tested this issue in your demo server of Magento which is
shared by Magento with you.
Now, this issue has been resolved in Magento ver 2.4.0.

------------------------------------
Thanks & Regards,
Rahul Rathour
Software Developer
Ksolves - https://www.ksolves.com
Email: rahul.rathour@ksolves.com
Contact: 8887846302
